SPECS/msgpack/msgpack.spec

```diff
@@ -0,0 +1,57 @@
+Summary:        MessagePack implementation for C and C++
+Name:           msgpack
```

Comment:
Please also update `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md ` for both packages.